packages/@react-types/shared/src/dna.d.ts

```diff
@@ -95,6 +99,8 @@ export type DimensionValue =
   | number;
 
 export type ColorValue =
+  | 'status'
+  | 'version'
```

Comment:
These seem like parsing issues. I don't think they are real variable names?

Reply:
ah you are right, I'll update the regex in the script